### PR TITLE
Added 'ftp' tag when publishing assets

### PR DESCRIPTION
**Pull request by zLegend-210**
#54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +23,11 @@ Add the package to your `composer.json` and run `composer update`.
 
 Add the service provider in `config/app.php`:
 
-    'Anchu\Ftp\FtpServiceProvider',
+    Anchu\Ftp\FtpServiceProvider::class,
```
Comment (harishanchu) [inline README.md]: I think this won't work for everyone.
The special ::class constant are available as of PHP 5.5.0 but laravel 5.0 supports PHP >= 5.4.

Comment (zLegend-210) [inline README.md]: okay

**Comment by zLegend-210** [comment]
The tag name normally is the package name or the file name, 'config' is pretty common tag, I think we should specify that to the package so I suggest 'ftp' or what you think ?

**Comment by harishanchu** [comment]
As per laravel package [development documentation](https://laravel.com/docs/5.0/packages#publishing-file-groups) tag names are to group files.

If you just want to publish resources of just a package you can use this syntax: 

> php artisan vendor:publish --provider="Vendor\Providers\PackageServiceProvider" --tag="config"

note the `provider` argument.

So right tag name for configuration resource is `config` isn't it?

**Comment by zLegend-210** [comment]
You're right.

File: README.md [hunk 2]
```diff
@@ -25,7 +27,7 @@ Add the service provider in `config/app.php`:
 
 Configuration
 ------------
-Run `php artisan vendor:publish` and modify the config file(`config/ftp.php`) with your ftp connections.
+Run `php artisan vendor:publish --tag=ftp` and modify the config file(`config/ftp.php`) with your ftp connections.
```
Comment (harishanchu) [inline README.md]: Hey, you missed to update Readme. Tag name is still ftp here.

**Comment by harishanchu** [comment]
Thanks buddy!

**Pull request merged**